encode/vp9enc.c

```diff
@@ -203,6 +203,22 @@ vp9enc_write_dword(char *ptr, uint32_t value)
     *(tmp + 3) = (value >> 24) & 0XFF;
 }
 
+static void
+vp9enc_write_qword(char *ptr, uint64_t value)
+{
+    uint8_t *tmp;
+
+    tmp = (uint8_t *)ptr;
+    *(tmp) = (value >> 0) & 0XFF;
+    *(tmp + 1) = (value >> 8) & 0XFF;
+    *(tmp + 2) = (value >> 16) & 0XFF;
+    *(tmp + 3) = (value >> 24) & 0XFF;
+    *(tmp + 4) = (value >> 32) & 0XFF;
+    *(tmp + 5) = (value >> 40) & 0XFF;
+    *(tmp + 6) = (value >> 48) & 0XFF;
+    *(tmp + 7) = (value >> 56) & 0XFF;
+}
+
```

Comment:
I know this is test app, but perhaps it is a good idea to merge vp9enc_write_qword(),  vp9enc_write_word() and vp9enc_write_dword() in a single function, with a loop and a defined halt condition as parameter.

Reply:
I just added  vp9enc_write_qword(), I didn't want to touch the rest of the code using vp9enc_write_word() and vp9enc_write_dword()